assets/js/blocks/active-filters/active-attribute-filters.js

```diff
@@ -46,6 +47,11 @@ const ActiveAttributeFilters = ( {
 
 	const attributeLabel = attributeObject.label;
 
+	const filteringForPhpTemplate = getSetting(
```

Comment:
Shall we update this to use `getSettingWithCoercion` so its consistent with [other filters](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/trunk/assets/js/blocks/attribute-filter/block.js#L66-L70)?